test/SILOptimizer/specialize_checked_cast_branch.swift

```diff
@@ -34,8 +34,6 @@ public func ArchetypeToArchetypeCast<T1, T2>(t1 : T1, t2 : T2) -> T2 {
 // CHECK:   return [[T0]]
 //
 // CHECK: bb2
-// CHECK:   integer_literal $Builtin.Int1, -1
-// CHECK:   cond_fail
```

Comment:
Please please the integer_literal+cond_fail with a check for the trap builtin:

// CHECK:   builtin "int_trap"

Reply:
Done